LiteDB/Client/Database/LiteDatabase.cs

```diff
@@ -32,19 +32,19 @@ public partial class LiteDatabase : ILiteDatabase
         /// <summary>
         /// Starts LiteDB database using a connection string for file system database
         /// </summary>
-        public LiteDatabase(string connectionString, BsonMapper mapper = null)
-            : this(new ConnectionString(connectionString), mapper)
+        public LiteDatabase(string connectionString, BsonMapper mapper = null, Action<EngineSettings> engineSettingsAction = null)
```

Comment:
this is a breaking change, so in order to release it in the SR I suggest to add an overload to this ctor accepting this new Action